components/PersonView/PersonView.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useEffect } from 'react';
+import { getResident } from 'utils/api/residents';
+
+const PersonView = ({ personId }) => {
+  const [person, setPerson] = useState();
+
+  const getPerson = async (personId) => {
+    try {
+      const data = await getResident(personId);
+      setPerson(data);
+    } catch {
+      console.log('error');
+    }
+  };
+  useEffect(() => {
+    getPerson(personId);
+  }, [personId]);
+
+  return (
+    <>
+      <h1>View person</h1>
+      <h2 className="personDetails">PERSON DETAILS</h2>
+      <hr className="personDetailsHr" />
+      <dl className="govuk-summary-list--no-border   lbh-summary-list">
```

Comment:
So we have `lbh-summary-list`? It seems we don't have any style attached to it 🤔

Reply:
Not sure...I copied it from lbh frontend. Take it out?

Reply:
We are not using the `lbh-frontend` library, for a series of reasons, we are referencing to https://design-system.service.gov.uk/components/.

Reply:
Ok I'll take the lbh things out.